### DISCUSSION TITLE
Dark mode theme query

### DISCUSSION THREAD

**Discussion by mokone91** [tailwindlabs/tailwindcss#10541] [Help]
Hi! is there any way to get dark-mode media query using theme() im css?

In my case i need to customize scrollbar with my colors, but i need to keep ability to change darkmode class(attribute) in theme without changing other styles

**Comment by wongjn** [comment]
In CSS, you could use the [native media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme):

```css
.foo {
  /* … */
}

@media (prefers-color-scheme: dark) {
  .foo {
    /* … */
  }
}
```

**Reply by mokone91** [reply]
I keep ability to override this method, some of project which use my theme can rewrite setting to next variants:
-  darkMode: ['class', '[data-theme="dark"]'] to not conflict with legacy code
-  darkMode: ['class'] for storybook, as it can't switch it by other variants
-   darkMode: ['class', 'mode-dark'] for https://github.com/rogden/tailwind-config-viewer

**Reply by mokone91** [reply]
its strange that we have 'dark:' modifier for classes, but nothing for css/sass
